src/material-experimental/theming/_custom-tokens.scss

```diff
@@ -653,6 +653,17 @@
   ));
 }
 
+/// Generates custom tokens for the mat-list.
+/// @param {Map} $systems The MDC system tokens
+/// @param {Boolean} $exclude-hardcoded Whether to exclude hardcoded token values
+/// @return {Map} A set of custom tokens for the mat-list
+@function list-active-indicator($systems, $exclude-hardcoded) {
```

Comment:
Should be called `list`, not `list-active-indicator` since the active indicator isn't a component.

Reply:
done